src/applications/gi-sandbox/containers/GiBillApp.jsx

```diff
@@ -78,13 +78,15 @@ export function GiBillApp({
           {constants.error && <ServiceError />}
           {!(constants.error || constants.inProgress) && (
             <DowntimeNotification appTitle={'GI Bill Comparison Tool'}>
-              <div className="tool-description">
-                <h1>GI Bill® Comparison Tool</h1>
-                <p className="vads-u-font-size--h3 vads-u-color--gray-dark">
-                  Use the GI Bill Comparison Tool to see how VA education
-                  benefits can pay for your education.
-                </p>
-              </div>
+              {!onProfilePage && (
```

Comment:
Since we only have a handful of pages, could you just make this into a component and include it on the pages where we want it?